src/pipx/commands/install.py

```diff
@@ -107,23 +106,6 @@ def install(
                 include_dependencies,
                 force=force,
             )
-        except FileNotFoundError as e:
-            venv.remove_venv()
-            if python in str(e) or "The system cannot find the file specified" in str(e):
-                print(
-                    pipx_wrap(
-                        f"""
-                           {emojis.hazard} No executable for the provided Python version '{python}' found.
-                           Please make sure the executable name is on your PATH /
-                           the path to the executable is correct.
-                        """,
-                        subsequent_indent=" " * 4,
-                    )
-                )
-                return EXIT_CODE_SPECIFIED_PYTHON_EXECUTABLE_NOT_FOUND
-            else:
-                print()
-                raise
```

Comment:
I have removed this block as it's now covered by the new check at the start of the application. I implemented it in https://github.com/pypa/pipx/pull/1171